out_writeapi.go

```diff
@@ -278,13 +303,13 @@ func FLBPluginFlushCtx(ctx, data unsafe.Pointer, length C.int, tag *C.char) int
 	// Look up through reference
 	config, ok := configMap[id]
 	if !ok {
-		log.Printf("Skipping flush because config is not found for tag: %s.", id)
-		return output.FLB_OK
+		log.Printf("Skipping flush because config is not found for tag: %d.", id)
+		return output.FLB_ERROR
 	}
 
-	responseErr := checkResponses(ms_ctx, config.results, false)
-	if responseErr == 1 {
-		log.Fatal("error in checking responses noticed in flush")
+	responseErr, currerr := checkResponses(ms_ctx, config.results, false)
```

Comment:
Probably can omit responseErr since we also return an err from checkResponses().